packages/react-html/README.md

```diff
@@ -230,6 +230,18 @@ Renders a `<link />` tag with the necessary props to specify a favicon. Accepts
 
 Renders a `<link />` tag that preconnects the browser to the host specified by the `source` prop. You can read more about preconnecting on [Google’s guide to resource prioritization](https://developers.google.com/web/fundamentals/performance/resource-prioritization#preconnect).
 
+### `<Baseline />`
```

Comment:
I don't really like this name, especially since we have the concept of `baseline` bundles in sewing-kit. Actually I'm not sure we want this to be a component. I sort of feel like the basic meta tags should just always be rendered by `HTML` and then `title` and `favicon` can either be used directly or passed as props to `HTML` itself.

Reply:
I like Mal's idea here 👍